NuRadioReco/modules/channelGalacticNoiseAdder.py

```diff
@@ -87,8 +98,13 @@ def begin(
         
         # initialise sky model
         if skymodel == 'lfmap':
-            sky_model = LFmap()
-            print("Using LFmap as sky model")
+            try:
+                sky_model = LFmap()
+                print("Using LFmap as sky model")
```

Comment:
Since you initialised a logger, why not replace all print statements with a proper logger.error() call? ;)